hack/install-e2e-dependencies.sh

```diff
@@ -13,6 +13,7 @@ minikube start \
      --vm-driver=kvm \
      --kubernetes-version="$KUBERNETES_VERSION" \
      --bootstrapper=kubeadm \
+     --memory 4096 \
```

Comment:
@wallrj this should hopefully solve some performance issues you've had with cassandra running in e2e tests

Reply:
Great. Although my builds have been quite reliable lately.